src/js/fluxcontroller.js

```diff
@@ -308,8 +308,13 @@ define(function (require, exports, module) {
                     return actionName[0] !== "_";
                 })
                 .forEach(function (actionName) {
-                    var action = actionModule[actionName],
-                        name = actionModuleName + "." + actionName,
+                    var action = actionModule[actionName];
```

Comment:
Paging Dr. @iwehrman, I believe this was made for `RENDITION_SIZE`... maybe that value should be exported in adapter/libraries then